### PR TITLE
Feature/reset sessions

### PR DESCRIPTION
**Pull request by peggles2**
Hacker 1 reported  https://hackerone.com/reports/514858
Session has to be reset after the user is authenticated

### CODE REVIEW

File: app/controllers/omniauth_callbacks_controller.rb [hunk 1]
```diff
@@ -6,7 +6,9 @@ def login_dot_gov
     @user = User.from_omniauth(auth)
     return unless @user.persisted?
 
-    UserSession.create(@user)
+    reset_session
```
Comment (MothOnMars) [inline app/controllers/omniauth_callbacks_controller.rb]: Please add a spec for this

File: app/controllers/omniauth_callbacks_controller.rb [hunk 2]
```diff
@@ -6,7 +6,9 @@ def login_dot_gov
...
+    @user_session = UserSession.create(@user)
```
Comment (MothOnMars) [inline app/controllers/omniauth_callbacks_controller.rb]: This shouldn't need to be assigned to an instance variable...?

**Comment by peggles2** [comment]
@MothOnMars this is ready for review

**Comment by peggles2** [comment]
@MothOnMars this is ready to review. Please review this before reviewing any other new pull requests. thanks!

**Pull request merged**